src/humancellatlas/data/metadata/helpers/staging_area.py

```diff
@@ -0,0 +1,334 @@
+from collections import (
+    defaultdict,
+)
+import json
+import logging
+import os
+from typing import (
+    ClassVar,
+    List,
+    Mapping,
+    Optional,
+    Sequence,
+    Set,
+    Tuple,
+    Union,
+)
+from uuid import (
+    UUID,
+    uuid5,
+)
+
+import attr
+from furl import (
+    furl,
+)
+from github import (
+    Github,
+    UnknownObjectException,
+)
+from github.Repository import (
+    Repository,
+)
+
+from humancellatlas.data.metadata.api import (
+    JSON,
+)
+from humancellatlas.data.metadata.helpers.exception import (
+    reject,
+    require,
+)
+from humancellatlas.data.metadata.helpers.schema_validation import (
+    SchemaValidator
+)
+
+logger = logging.getLogger(__name__)
```

Comment:
https://github.com/DataBiosphere/azul/blob/develop/CONTRIBUTING.rst#logging

This isn't legacy code so

```suggestion
log = logging.getLogger(__name__)
```